scripts/bootstrap.sh

```diff
@@ -31,6 +31,7 @@ if [[ "$OSTYPE" == "linux-gnu"* ]]
 then
   echo "Installing just in /usr/local/bin/just"
   install -m 755 "$PWD/bin/just-1.14.0-x86_64-unknown-linux-musl" /usr/local/bin/just
+  just --completions bash > /etc/bash_completion.d/just
```

Comment:
Will this take effect immediately in the shell I ran bootstrap from? Or does in need a new shell to activate?

Reply:
pretty sure it needs a new shell, same as changes to profile

Reply:
should we update docs for install procedure?

Reply:
updated